nilearn/datasets/func.py

```diff
@@ -2215,6 +2215,15 @@ def fetch_language_localizer_demo_dataset(data_dir=None, verbose=1):
 def fetch_bids_langloc_dataset(data_dir=None, verbose=1):
     """Download language localizer example :term:`bids<BIDS>` dataset.
 
+    .. deprecated:: 0.10.2
```

Comment:
```suggestion
    .. deprecated:: 0.11.0
```
as 0.10.2 is already released